app/src/main/java/org/jellyfin/androidtv/ui/startup/SplashFragment.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.ui.startup
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.jellyfin.androidtv.R
+
+class SplashFragment : Fragment() {
+	override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+		return inflater.inflate(R.layout.fragment_startup, container, false)
+	}
+}
```

Comment:
You can pass the layout-id to fragments and use the default `onCreateView` function like so:

```suggestion
class SplashFragment : Fragment(R.layout.fragment_startup)
```

Reply:
:+1: Should be fixed in any classes added here.